pigskin.py

```diff
@@ -406,22 +407,72 @@ def parse_shows(self):
             episodes_data = self.make_request(episodes_url, 'get')['modules']['archive']['content']
 
             # 'season' is often left unset. It's impossible to know for sure,
-            # but the year of the broadcast date seems like a sane best guess.
-            # TODO: but apparently 'scheduleDate' often contains errors. Yay...
+            # but the year of the current Season seems like a sane best guess.
```

Comment:
IIRC, ``current_season`` will be the current ongoing season (2018). If so, then I think ``scheduleDate`` would be more likely to be accurate than ``current_season`` for old archived shows. Or am I missing something?

Reply:
current_season is the whole time 2018 (for the current season).
scheduleDate could also be 2019 for the current season so its safer to use current season.

Reply:
@kaileu Ahh. Ok; if that's the case, then this approach makes sense. That would be worth explaining as a comment in the code.